recipes/haystack_bio/build.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+$PYTHON setup.py install
```

Comment:
Can you please use:
`$PYTHON setup.py install --single-version-externally-managed --record=record.txt`